spark-derived-feature-view/docker-compose.yaml

```diff
@@ -0,0 +1,73 @@
+version: "2.1"
+services:
+  spark-master:
+    image: feathub-spark:3.3.1
+    container_name: spark-master
+    ports:
+      - "8080:8080"
+      - "7077:7077"
+    depends_on:
+      - datanode
+    environment:
+      - SPARK_LOCAL_IP=spark-master
+      - SPARK_WORKLOAD=master
+
+  spark-worker-a:
+    image: feathub-spark:3.3.1
+    container_name: spark-worker-a
+    ports:
+      - "8081:8080"
+      - "7000:7000"
+    depends_on:
+      - spark-master
+    environment:
+      - SPARK_MASTER=spark://spark-master:7077
+      - SPARK_WORKER_CORES=1
+      - SPARK_WORKER_MEMORY=1G
+      - SPARK_DRIVER_MEMORY=1G
+      - SPARK_EXECUTOR_MEMORY=500M
+      - SPARK_WORKLOAD=worker
+      - SPARK_LOCAL_IP=spark-worker-a
+    volumes:
+       - ./apps:/tmp/spark-apps
+
+  spark-worker-b:
+    image: feathub-spark:3.3.1
+    container_name: spark-worker-b
+    ports:
+      - "8082:8080"
+      - "7001:7000"
+    depends_on:
+      - spark-master
+    environment:
+      - SPARK_MASTER=spark://spark-master:7077
+      - SPARK_WORKER_CORES=1
+      - SPARK_WORKER_MEMORY=1G
+      - SPARK_DRIVER_MEMORY=1G
+      - SPARK_EXECUTOR_MEMORY=500M
+      - SPARK_WORKLOAD=worker
+      - SPARK_LOCAL_IP=spark-worker-b
+
+  namenode:
+    image: bde2020/hadoop-namenode:1.1.0-hadoop2.7.1-java8
```

Comment:
The use of Hadoop is independent of the use of DerivedFetureView. The example execution time would be longer due to the use of Haddop. And the README for this example also becomes more complicated than necessary.

Also note that the existing Flink DerivedFeatureView example does not use Hadoop. And we have dedicated flink-read-write-hdfs to test how to use Hadoop with Flink.

For performance reasons and consistency, should we just use local file system in this example?